packages/backend/src/open_payments/payment_pointer/service.ts

```diff
@@ -12,15 +16,27 @@ import {
 } from './model'
 import { BaseService } from '../../shared/baseService'
 import { AccountingService } from '../../accounting/service'
+import { PaymentPointerStatus } from '../../graphql/generated/graphql'
```

Comment:
The layers (GraphQL & business logic layer) are being coupled here (since its a GraphQL import).
ie, other services that would want to use this method shouldn't have to import a GraphQL type.

We can update the `UpdateOptions` type to fix this, see below: